snuba/cli/subscriptions.py

```diff
@@ -119,9 +119,12 @@ def subscriptions(
     dataset = get_dataset(dataset_name)
 
     storage = dataset.get_default_entity().get_writable_storage()
+    assert storage is not None
     storage_key = storage.get_storage_key()
 
     loader = enforce_table_writer(dataset).get_stream_loader()
+    topic_spec = loader.get_commit_log_topic_spec()
+    assert topic_spec is not None
```

Comment:
All datasets are supposed to have a writable storage, we should be able to make that non nullable. I think we should fix it there and leave this as it is. If we add the assertion here we are hiding the issue.

Reply:
Just to clarify, this is a field on the KafkaStreamLoader, not the loader itself. And we actually pass None into that field sometimes: https://github.com/getsentry/snuba/blob/master/snuba/datasets/table_storage.py#L122

Reply:
you are right on this, but this comment also addresses line 122. Every dataset should have a writable storage.